File: models/usuario.py
class Usuario:
    def __init__(self, nome, email, dataNascimento, senha, id_usuario=None):
        self.id_usuario = id_usuario
        self.nome = nome
        self.email = email
        self.dataNascimento = dataNascimento
        self.senha = senha

    def __str__(self):
        return f"Usuario(id_usuario={self.id_usuario}, nome='{self.nome}', dataNascimento='{self.dataNascimento}')"

File: frontend/src/model/usuario.ts
export class Usuario {
    idUsuario?: number;
    nome?: string;
    email: string;
    dataNascimento?: string;
    senha: string;

    constructor(email: string, senha: string, nome?: string,  dataNascimento?: string, idUsuario?: number) {
        this.idUsuario = idUsuario;
        this.nome = nome;
        this.email = email;
        this.dataNascimento = dataNascimento;
        this.senha = senha;
    }
}